src/solver.jl

```diff
@@ -2,30 +2,88 @@
 #  This Source Code Form is subject to the terms of the Mozilla Public
 #  License, v. 2.0. If a copy of the MPL was not distributed with this
 #  file, You can obtain one at http://mozilla.org/MPL/2.0/.
-#===================================================
- This file implements the default PajaritoSolver
-===================================================#
-@enum DisaggSOC _disagg_soc_default _disagg_soc_on _disagg_soc_off
+
+#=========================================================
+This file implements the default PajaritoSolver
+=========================================================#
 
 export PajaritoSolver
+
 immutable PajaritoSolver <: MathProgBase.AbstractMathProgSolver
-    verbose::Int                # Verbosity level flag
-    algorithm                   # Choice of algorithm: "OA" or "BC"
-    mip_solver                  # Choice of MILP solver
-    cont_solver                 # Choice of Conic solver
-    opt_tolerance               # Relatice optimality tolerance
-    time_limit                  # Time limit
-    profile::Bool               # Performance profile switch
-    disaggregate_soc::DisaggSOC # SOCP disaggregator for SOC constraints
-    instance::AbstractString    # Path to instance
-    enable_sdp::Bool            # Indicator for enabling sdp support
-    force_primal_cuts::Bool     # Enforces primal cutting planes under conic solver
-    dual_cut_zero_tol::Float64  # Tolerance to check if a conic_dual is zero or not
+    path::AbstractString        # File path
+    log_level::Int              # Verbosity level flag
+    branch_cut::Bool            # Use BC algorithm, else use OA
```

Comment:
algorithm is a little confusing because we have many algorithms depending on what other options you select. do we project cuts, do we add SOC cuts, do we disaggregate. with the option branch_cut it's clear what you're choosing